api/locations/location.mock.ts

```diff
@@ -0,0 +1,37 @@
+/* tslint:disable:max-line-length */
+import MockAdapter from 'axios-mock-adapter';
+import {HttpMockAdapter} from '../common/httpMockAdapter';
+import {ALL_COURT_LOCATIONS} from "./data/location.mock.data";
+
+export const init = () => {
+  const mock: MockAdapter = HttpMockAdapter.getInstance();
+
+  const getHearingsUrl = /http:\/\/rd-professional-api-aat.service.core-compute-aat.internal\/refdata\/location\/court-venues\/venue-search\?search-string=[\w]*/;
+
+  mock.onGet(getHearingsUrl).reply(config => {
+    const requestURL = config.url;
+    const searchTermStartIdx = config.url.indexOf('=');
+    const searchTermEndIdx = config.url.indexOf('&');
+    let searchTerm;
+    if (searchTermEndIdx) {
```

Comment:
Same thing

Reply:
not necessary but I have changed it as you like